spec/lib/mercury/mercury_spec.rb

```diff
@@ -29,7 +29,7 @@
       em do
         Mercury.open do |m|
           m.close do
-            expect { m.publish(queue, {'a' => 1}) }.to raise_error /closed/
+            expect { m.publish(queue, 'a' => 1) }.to raise_error /closed/
```

Comment:
I like not seeing the braces for arguments when the hash is the last argument in the method call.